frontend/src/utils/mapStyles.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "featureType": "poi.business",
+    "stylers": [
+      {
+        "visibility": "off"
+      }
+    ]
+  }
```

Comment:
Preferably put that under `/src/shared/` instead of making another directory for these things. :)

Reply:
Sure I'll put it there!